packages/account/src/Containers/proof-of-ownership/card.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Button, Icon, Text } from '@deriv/components';
+import ExpandedCard from './expanded-card';
+import { TPaymentMethodInfo } from '../../Types';
+
+type TCardProps = {
+    details: TPaymentMethodInfo;
+    index: number;
+};
+
+/**
+ * Renders Chevron Icon which is used to expand card
+ * @name ExpansionIcon
+ * @param is_open - status to check if card is open
+ * @returns React Component
+ */
+const ExpansionIcon = ({ is_open }: { is_open: boolean }) => (
+    <Icon
+        icon='IcChevronUpBold'
+        className={classNames('proof-of-ownership__card-item-icon', {
+            'proof-of-ownership__card-item-icon--invert': !is_open,
+        })}
+    />
+);
+
+/**
+ * Renders payment method
+ * @name Card
+ * @param details - payment method details
+ * @param index - index of payment method
+ * @returns React Component
+ */
+const Card = ({ details, index }: TCardProps) => {
+    const [is_open, setIsOpen] = React.useState(false);
+
+    const onClickHandler = () => {
+        setIsOpen(!is_open);
+    };
+
+    return (
+        <div
+            className={classNames('proof-of-ownership__card', { 'proof-of-ownership__card-open': is_open })}
+            data-testid={details?.payment_method}
+            role='card-item'
+        >
+            <div className='proof-of-ownership__card-item' onClick={onClickHandler}>
+                <Icon
+                    icon={details?.icon || ''}
```

Comment:
cant we check if details?.icon and then render Icon component instead of passing ''